assets/flake.hbs

```diff
@@ -0,0 +1,91 @@
+# This flake was initially generated by the `fh` tool for FlakeHub
```

Comment:
Final nit:

could we embed the version of `fh` here (like we do in the installer)?